client/router_update.go

```diff
@@ -434,7 +434,7 @@ func (cli *VanClient) RouterUpdateVersionInNamespace(ctx context.Context, hup bo
 	}
 
 	if addCertsSharedVolume {
```

Comment:
Do we need to add another shared volume (the one related to the flow controller) in case the user updates Skupper?

Reply:
Yes. The next update will include update handling that will setup the sidecar and volume/mounts for the collector.